### PR TITLE
added sheldon week5 hw

### PR DESCRIPTION
**Pull request by Yunicorn228**

### CODE REVIEW

File: week5/sheldon/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// --------------------------------------first question
+// const isString = str => {
+//   if (str === +str) {
+//     return false;
+//   } else {
```
Comment (mxdi9i7) [inline week5/sheldon/index.js]: you can simplify this conditional statement, you don't need the else

File: week5/sheldon/index.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+//   }
+// };
+// console.log(isString("jiaolian"));
+// console.log(isString("123"));
+
+//--------------------------------------second question
+// const isBlank = str => {
+//   if (str == "") {
+//     return true;
+//   } else {
```
Comment (mxdi9i7) [inline week5/sheldon/index.js]: same as above

File: week5/sheldon/index.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+//   return str
+//     .toLowerCase()
+//     .split(" ")
+//     .join("-");
+// };
+// console.log(stringParameterize("MY Javascrpit SKIll is good af"));
+
+//--------------------------------------fifth question
+// const camelize = str => {
+//   //   return str.charAt(0).toUpperCase() + str.slice(1, str.length);
```
Comment (mxdi9i7) [inline week5/sheldon/index.js]: remove

File: week5/sheldon/index.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+// console.log(randomize(10, 15));
+
+// //--------------------------------------seventh question
+// const strip = str => {
+//   return str.trim();
+// };
+// console.log(strip(" 123 15212 200 "));
+
+// const striper = str => {
+//   if (str[0] === " " && str[length - 1] === " ") {
```
Comment (mxdi9i7) [inline week5/sheldon/index.js]: length undefined error

File: week5/sheldon/index.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+//   );
+// };
+
+// const formatPhoneNumber = input => {
+//   //用户输入信息
+//   const phoneArr = input.split("");
+//   //转化为arr
+//   const phoneNum = [];
+//   //用来收集号码
+//   for (var i = 0; i < phoneArr.length; i++) {
```
Comment (mxdi9i7) [inline week5/sheldon/index.js:107]: try not to use for loop

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]

File: week5/sheldon/index.js (hunk 1 above)
Comment (Yunicorn228) [reply week5/sheldon/index.js]: changed to return true

File: week5/sheldon/index.js (hunk 2 above)
Comment (Yunicorn228) [reply week5/sheldon/index.js]: changed to return false

File: week5/sheldon/index.js [hunk 6]
```diff
@@ -54,6 +51,14 @@
 // };
 // console.log(camelize("shelDon love coding"));
 
+// const camelize = str => {
+//   let wordArr = str.split(" ");
+//   wordArr[1] =
+//     wordArr[1][0].toUpperCase() + wordArr[1].slice(1, wordArr[1].length);
+//   return wordArr.join("");
+// };
```
Comment (mxdi9i7) [inline week5/sheldon/index.js:59]: have you tested this? looks like you are missing wordArr[0]

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]

**Review by mxdi9i7 — APPROVED** [review]

**Pull request merged**